tests/test_cli.py

```diff
@@ -149,25 +148,29 @@ def test_end_to_end_armormaxsize(capsys, tmp_path):
   assert not cap.out
 
 
+def test_end_to_end_large_file(capsys, tmp_path):
+  from covert.__main__ import main
+  import sys
+  fname = tmp_path / "test.dat"
+  outfname = tmp_path / "crypto.covert"
 
   # Write file with size too large for --armor
-  f = open(f"{str(fname)}", "wb")
-  f.seek(42505855)
-  f.write(b"\0")
-  f.close()
+  with open(f"{fname}", "wb") as f:
+    f.seek(42505855)
+    f.write(b"\0")
 
   # Try encrypting without -o
   sys.argv = f"covert -ea --password verytestysecret".split() + [ str(fname) ]
-  ret = main()
+  with pytest.raises(ValueError) as excinfo:
+    main()
+    assert "How about -o FILE to write a file?" in str(excinfo.value)
   cap = capsys.readouterr()
-  assert not ret
   assert not cap.out
-  assert "Too much data for console. How about -o FILE to write a file?" in cap.err
 
   # Try encrypting with -o
-  sys.argv = f"covert -e --password verytestysecret -ao".split() + [ str(outfname), str(fname) ]
-  ret = main()
+  sys.argv = f"covert -ea --password verytestysecret -o".split() + [ str(outfname), str(fname) ]
+  with pytest.raises(ValueError) as excinfo:
+    main()
+    assert "The data is too large for --armor." in str(excinfo.value)
   cap = capsys.readouterr()
-  assert not ret
-  assert not cap.out
-  assert "The data is too large for --armor." in cap.err
\ No newline at end of file
+  assert not cap.out
```

Comment:
Final newline missing.

Configure your editor to always add one at the end so you don't have to do it manually. Most editors have a setting for this, as well as removing any extra empty lines that there might be at the end. I believe I also saw an indent (a few spaces) on an otherwise empty line in some prior PR of yours, so also check that setting (remove whitespace at the end of lines).

These settings are important because otherwise we see in the repository some unnecessary changes where nothing but the spaces and newlines are modified. CLI `git diff` shows warnings for them (as might a GitLens or similar feature inside your editor if it has one) and Github shows these ⛔ signs (probably not visible on this comment but you can see it if you look at the full diff).